frontend/src/Utils/Loader.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
-import { Bullseye, Spinner } from '@patternfly/react-core';
+import { Bullseye } from '@patternfly/react-core/dist/dynamic/layouts/Bullseye';
```

Comment:
@rohitkrai03 I believe we delivered a change that does this for us already at build time: https://github.com/patternfly/patternfly-react/pull/9298 - though we should confirm.

Reply:
I had to make this change because there were lint errors without this.

Reply:
I think this is coming from `forbid-pf-relative-imports` rule in `@redhat-cloud-services/eslint-config-redhat-cloud-services`.